dCommon/eUnequippableActiveType.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#ifndef __EUNEQUIPPABLEACTIVETYPE__H__
+#define __EUNEQUIPPABLEACTIVETYPE__H__
+
+#include <cstdint>
+
+enum class eUnequippableActiveType : int32_t {
```

Comment:
I think we should create another folder in dCommon for enums, this feels it could get excessive really quickly.

Reply:
Will do that in a separate PR